tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -315,10 +348,19 @@ def _directory_factory(
             nlb_stack_name = created_directory_stacks.get(region, {}).get("nlb")
             logging.info("Using NLB stack named %s created by another test", nlb_stack_name)
         else:
-            certificate_arn = _generate_certificate()
+            common_name = "ldap.simplead.multiuser.pcluster"
```

Comment:
The common name should not be hard wired to simpleAD because we should use the same approach also for MsAD.

Reply:
What would be the alternative to hard-coding?

Reply:
for instance:

`common_name = f"{directory_type}.multiuser.pcluster"`

Reply:
I think the `ldap` prefix is not strictly required to characterize the endpoint because both SimpleAd and MsAD are always LDAP